lib/helpers/custom_transitions.dart

```diff
@@ -24,12 +24,32 @@ class CustomTransitionsBuilders {
     if (animation.status == AnimationStatus.reverse) {
       curvedAnimation = CurvedAnimation(parent: animation, curve: Curves.easeInExpo);
     }
+    double? dragDistance;
```

Comment:
Use a static property or something in this class if this function runs often.

Reply:
It's running a few times based on the level the screen is in. So, when a page is opened from home, it is called twice. When another page is called from there, it is called three times, etc.